client/checkout/classic/event-handlers.js

```diff
@@ -166,48 +166,37 @@ jQuery( function ( $ ) {
 
 		for ( const method of paymentMethodsKeys ) {
 			if ( bnplMethods.includes( method ) ) {
-				const targetLabel = document.querySelector(
-					`label[for="${ labelBase }${ method }"]`
-				);
 				const containerID = `stripe-pmme-container-${ method }`;
+				const container = document.getElementById( containerID );
 
-				if ( document.getElementById( containerID ) ) {
-					document.getElementById( containerID ).innerHTML = '';
+				if ( ! container ) {
+					continue;
 				}
 
-				if ( targetLabel ) {
```

Comment:
Changing this slightly - the container will always be present (generated from the backend), the JS will use it to mount the BNPL messaging.